docs/section4/4-3.md

```diff
@@ -0,0 +1,39 @@
+# Airside System – Rule 4-3
+
+**Rule ID:** 4-3
+ 
+**Rule Description:** Schedules for HVAC fans that provide outdoor air for ventilation shall run continuously whenever spaces are occupied.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules for the proposed building excluding exception #1. 
+**Appendix G Section Reference:** None
+**Data Lookup:** None 
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+
+**Applicability Checks:** 
+
+1. This rule does not apply when no heating and/or cooling system is to be installed, and a heating or cooling system is being simulated only to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d.
+ 
+**Function Call:** 
+
+1. get_HVAC_systems_applicable_10cd
+
+**Manual Check:** None  
+
+**Rule Logic:**
+**Applicability Check 1:** 
+- Call function to get list of inapplicable HVAC systems: `inapplicable_HVAC_systems_list_p = get_HVAC_systems_applicable_10cd(U_RMR,P_RMR)`
+- For each HeatingVentilationAirAconditioningSystem in the P_RMR check if this check is applicable: `for each hvac_p in P_RMR...hvac_systems:`
+    - Check if hvac_p is not in the list of inapplicable systems, if it is not found then set applicability flag to true: `if hvac_p not in inapplicable_HVAC_systems_list_p: rule_applicability_flag = TRUE`   
+    - If rule_applicability_flag = TRUE for hvac_p: `if rule_applicability_flag == TRUE:`
+        - For each fan system in hvac_p: `fan_systems_p in hvac_p.fan_systems:`
+            - Get operation during occupied periods: `operation_during_occupied_p = fans_systems_p.operation_during_occupied`
+            - Get minimum_outdoor_airflow: `minimum_outdoor_airflow_p = fans_systems_p.minimum_outdoor_airflow`
+
+            **Rule Assertion:**
+            - Case 1: For each applicable fan system, if fan system operation during occupied periods is continuous and the minimum OA is greater than 0 in the P_RMR: `if operation_during_occupied_p == “CONTINUOUS” and minimum_outdoor_airflow_p >0 : PASS`
```

Comment:
Does this align with the approach that we agreed on for providing outcome on individual component level vs. rule level?

Reply:
Yes